### PR TITLE
Finish project generation of spring cloud azure

### PR DESCRIPTION
**Pull request by Incarnation-p-lee**

### CODE REVIEW

File: src/main/java/com/microsoft/azure/springcloudplayground/generator/ProjectGenerator.java [hunk 1]
```diff
@@ -20,6 +20,8 @@
 import org.springframework.util.Assert;
 import org.springframework.util.FileSystemUtils;
 import org.springframework.util.StreamUtils;
+import org.springframework.util.StringUtils;
+import retrofit2.http.HEAD;
```
Comment (sophiaso) [inline src/main/java/com/microsoft/azure/springcloudplayground/generator/ProjectGenerator.java]: Seems this `HEAD` not used?

Comment (Incarnation-p-lee) [reply src/main/java/com/microsoft/azure/springcloudplayground/generator/ProjectGenerator.java]: yeah，should remove this.

Comment (Incarnation-p-lee) [reply src/main/java/com/microsoft/azure/springcloudplayground/generator/ProjectGenerator.java]: It is somewhat wired from IDE, as I enabled `auto import`, should remove this automatically.

File: src/main/resources/templates/docker/docker-compose.yml [hunk 2]
```diff
@@ -4,48 +4,48 @@ services:
   config-server:
     image: "demo/cloud-config-server"
     ports:
-      - "{{cloud-config-server-port}}:{{cloud-config-server-port}}"
+      - "{{port}}:{{port}}"
```
Comment (sophiaso) [inline src/main/resources/templates/docker/docker-compose.yml]: Does that mean all services are using same `{{port}}`?

Comment (Incarnation-p-lee) [reply src/main/resources/templates/docker/docker-compose.yml]: yes, but there are kinds of different `Map` represent microservices, for each one has entry `port`.

**Pull request merged**